doc/source/theory/multiphysics/electromagnetism/time_harmonic_weak_form.rst

```diff
@@ -0,0 +1,41 @@
+=============================================================
+Electromagnetic Wave Finite Element Formulation
+=============================================================
+
+This section describes the FEM formulation of the time-harmonic Maxwell's equations. For further details on the subject, the reader is referred to the following references: `Larson <https://books.google.ca/books/about/The_Finite_Element_Method_Theory_Impleme.html?id=Vek_AAAAQBAJ&redir_esc=y>`_ and `Jian-Ming <https://www.wiley.com/en-br/The+Finite+Element+Method+in+Electromagnetics%2C+3rd+Edition-p-9781118571361>`_ . 
+
+Starting from the strong form of the electromagnetic field  --- Note that it is not necessary to solve both the magnetic and the electric field wave equation since one results from the other and one can choose the one that suits the best. --- showed in the :doc:`Time Harmonic Maxwell's Equations <time_harmonic>` section :
```

Comment:
```suggestion
Starting from the strong form of the electromagnetic field  --- Note that it is not necessary to solve both the magnetic and the electric field wave equation since one results from the other and one can choose the one that suits the best. --- showed in the :doc:`Time Harmonic Maxwell's Equations <time_harmonic>` section:
```